test/tests/trackers.js

```diff
@@ -0,0 +1,71 @@
+(function() {
+  QUnit.module("trackers");
+
+  var trackers = require('trackers');
+  var settings = require('settings');
+
+  var basicBlocking = [
+    { 'url': 'https://doubleclick.net', 'block': true},
+    { 'url': 'https://duckduckgo.com', 'block': false}
+  ];
+
+  basicBlocking.forEach(function(test) {
+      QUnit.test("block url", function (assert) {
+          settings.updateSetting('extensionIsEnabled', true);
+          var toBlock = trackers.isTracker(test.url, '', 0);
+          toBlock = toBlock ? true : false;
+          assert.ok(toBlock === test.block, 'url should be blocked');
+      });
+  });
+
+  basicBlocking.forEach(function(test) {
+      QUnit.test("turn off blocking", function (assert) {
+          settings.updateSetting('extensionIsEnabled', false);
+          var toBlock = trackers.isTracker(test.url, '', 0);
+          toBlock = toBlock ? true : false;
+          assert.ok(toBlock === false, 'url should not be');
+      });
+  });
+
+  var thirdPartyTests = [
+      { 'url': 'https://facebook.com', 'host': 'https://facebook.com', 'block': false, 'message': 'don\'t block first party requests'},
+      { 'url': 'https://facebook.com', 'host': 'https://reddit.com', 'block': true, 'message': 'should block third party request'},
+      { 'url': 'https://facebook.com', 'host': 'https://instagram.com', 'block': false, 'message': 'should not block third party requests owned by same parent company'}
+  ];
+
+  thirdPartyTests.forEach(function(test) {
+      QUnit.test("third party blocking", function (assert) {
+          settings.updateSetting('extensionIsEnabled', true);
+          settings.updateSetting('socialBlockingIsEnabled', true);
+          var toBlock = trackers.isTracker(test.url, test.host, 0);
+          toBlock = toBlock ? true : false;
+          assert.ok(toBlock === test.block, test.message);
+      });
+  });
+
+  var socialBlocking = [
```

Comment:
We should probably leave out social blocking tests, since we're pushing off this feature. Doesn't matter for this PR though